### DISCUSSION TITLE
Is it possible server to support distributed CPUs inference?

### DISCUSSION THREAD

**Discussion by hyperbolic-c** [ggml-org/llama.cpp#6790] [Q&A]
Just like distributed CPU training, for example, [mindspore distributed cpu train](https://www.mindspore.cn/tutorials/experts/en/r2.0/parallel/train_cpu.html), any solution about LLM inference ? Implement inference on multiple device nodes with multiple CPUs? thanks！！

**Comment by kannon92** [comment]
In the repo, there is support for MPI.
